packages/next/client/components/noop-head.tsx

```diff
@@ -0,0 +1,11 @@
+import { warnOnce } from '../../shared/lib/utils/warn-once'
+
+if (process.env.NODE_ENV !== 'production') {
+  warnOnce(
```

Comment:
This should warn as part of running the `NoopHead` render because it could be a component rendering `next/head` is rendered in both `app` and `pages`.